installing/installing_bare_metal/bare-metal-using-bare-metal-as-a-service.adoc

```diff
@@ -0,0 +1,41 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="bare-metal-using-bare-metal-as-a-service"]
+= Using bare metal as a service
+include::_attributes/common-attributes.adoc[]
+:context: bare-metal-using-bmaas
+
+toc::[]
+
+{product-title}'s Bare Metal as a Service (BMaaS) feature enables you to provision and manage bare-metal hosts by using the Metal^3^ API. These hosts, external to the {product-title} cluster, can run workloads that might not be suitable for containerisation or virtualisation, such as applications requiring direct hardware access, high-performance computing tasks or legacy applications. BMaaS provides the following capabilities:
+
+* Provisioning of bare-metal hosts, including initial configuration.
+* Lifecycle management, such as power management, firmware updates, and decommissioning.
+
+As standalone systems, these hosts operate independently of the {product-title} cluster and support diverse workloads by integrating bare-metal resources with containerized and virtualized applications. BMaaS can run other operating systems, but only {op-system-base-full} and CentOS Stream 9 underwent testing.
```

Comment:
```suggestion
As standalone systems, these hosts operate independently of the {product-title} cluster and support diverse workloads by integrating bare-metal resources with containerized and virtualized applications. BMaaS can run other operating systems, but only {op-system-base-full} and CentOS Stream 9 were tested.
```
Hmm, then why do we give instructions later for Fedora? Is Fedora supported by the Support team? Is CentOS? Or just RHEL?